File: Uri/1043.py
x = raw_input().split(' ')
a = float(x[0])
b = float(x[1])
c = float(x[2])

if ((abs(a-b)) < c and c < (a + b)):
    p = a+b+c
    print "Perimetro = %.1f"%p
else:
    ar = (a+b)*c/2
    print "Area = %.1f"%ar

File: Uri/1043.c
#include<stdio.h>
#include<stdlib.h>
#include<math.h>

int main(){
    float x, y, z, p, a;
    scanf("%f %f %f", &x, &y, &z);
    if ((abs(x-y)) < z && z < (x + y)){
        p = x + y + z;
        printf("Perimetro = %.1f\n", p);

    }
    else{
        a = ((x + y)*z)/2;
        printf("Area = %.1f\n", a);

    }
    return 0;
}
